README.md

```diff
@@ -1 +1,133 @@
-# @lemoncode/react-image-focal-point
+<div align="center">
+  <h1>React Image Focal Point</h1>
+  <a href="https://lemoncode.github.io/react-image-focal-point">
+    <img
+      height="100"
+      width="100"
+      alt="logo"
+      src="https://raw.githubusercontent.com/lemoncode/react-image-focal-point/develop/website/static/img/logo.svg"
+    />
+  </a>
+
+  <br/>
+
+  <p>You can just drag and drop a focal point button on top of your image and enhance your cropped images.</p>
```

Comment:
Let the user set the picture focal point just by visually dragging and dropping on top of the selected image

Reply:
We have to add an animated gif, picture with focal on the left side, picture on the right side and move stuff